Assets/LeapMotionModules/UI/Scripts/Constraints/Abstract/Linear2Constraint.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Leap.Unity.Attributes;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.UI.Constraints {
+
+  /// <summary>
+  /// Two-dimensional linear constraint.
+  /// </summary>
+  public abstract class Linear2Constraint : PositionConstraint {
+
+    [ConstraintDebugging]
+    [SerializeField]
+    [Range(0, 1)]
+    [Disable] // (Readonly)
+    protected Vector2 _uvCoords = Vector2.zero;
```

Comment:
Two dimensional coordinates from 0 to 1. In other words, the same thing UV coordinates usually represent, which is why I named them uvCoords.